templates/metrics.html

```diff
@@ -9,18 +9,16 @@
 {% block content %}
 <div class="superAbout">
     {% if not request.user_agent.is_mobile %}
-        {% include '_sidebar.html' with whichPage='metrics' title="Metrics" heTitle="מדדים" %}
+        {% include '_sidebar.html' with whichPage='metrics' %}
     {% endif %}
     <div id="metricsPage" class="biReady container static">
 
 
         <div class="inner">
-            {% if request.user_agent.is_mobile %}
                 <h1 class="mobileAboutHeader">
```

Comment:
@stevekaplan123 
Regarding your comment in the description, I would agree that if we're now showing the `aboutHeader` on non-mobile as well, then we should change the class name :-)